modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase

```diff
@@ -2116,24 +2116,25 @@ definition {
 
 		ObjectCustomValidation.gotoTab(tabName = "Conditions");
 
-		ObjectCustomValidation.insertScriptOnValidation(scriptValidation = "invalidFields = (customField == 'Decline Entry')");
+		ObjectCustomValidation.insertScriptOnValidation(scriptValidation = "invalidFields = (customField == 'Entry Decline')");
 
 		ObjectCustomValidation.addErrorMessage(errorMessage = "This entry is not possible.");
 
 		PortletEntry.save();
 
-		for (var entryTitle : list "Entry Test,Decline Entry") {
-			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
-				fieldName = "customField",
-				objectName = "CustomObject153550",
-				value = ${entryTitle});
-		}
-
 		ObjectAdmin.goToCustomObject(objectName = "CustomObject153550");
 
+		for (var entryName : list "Entry Test,Entry Decline") {
+			ObjectPortlet.addSingleFieldEntryViaUI(entry = ${entryName});
+
+			PortletEntry.save();
```

Comment:
This `PortletEntry.save();` is not necessary, because the macro `addSingleFieldEntryViaUI` already made it.